components/Header/InlineMenu.js

```diff
@@ -0,0 +1,38 @@
+import style from "./style.scss";
+import Link from "next/link";
+
+const InlineMenu = () => (
+  <div className={style.inlineMenu}>
+    <div className={style.menu}>
+      <Link href="/speak-up">
```

Comment:
Wydaje mi się, że `<Link>` powinien mieć tylko jedno dziecko - `<a>` - bo przekazuje do komponentu niżej atrybut `href`. Jak zamienisz `<Link>` i `<a>` to nie powinnaś potrzebować stylu `cursor: pointer;`.